src/main/content/_assets/js/blog.js

```diff
@@ -86,38 +131,51 @@ var blog = function(){
 
     $(window).on('popstate', function(){
         removeFilter();
-        var tag = getTagFromUrl();
-        if (tag) {
-            filterPosts(tag);
+        var tagList = getTagFromUrl();
+        if (tagList.length > 0) {
+            filterPosts(tagList);
         }
     });
 
     function getTagFromUrl(){
-        var tag;
+        var tList = [];
```

Comment:
small nit pick
Can `tList` be `tagList` to improve human readability?

Reply:
I know sometimes JavaScript linters are unaware of the method scoping of variables and will sometimes flag using the same variable name in different methods. I am wondering maybe that is why you used `tList`?